templates/auth_gssapi.conf.erb

```diff
@@ -0,0 +1,14 @@
+
+<Location /users/extlogin>
+  SSLRequireSSL
+  AuthType GSSAPI
+  AuthName "GSSAPI Single Sign On Login"
+  GssapiCredStore keytab:<%= scope.lookupvar('::foreman::http_keytab') %>
```

Comment:
Since we're Puppet 4 only, `::` prefix is no longer needed since everything is by definition absolute
```suggestion
  GssapiCredStore keytab:<%= scope.lookupvar('foreman::http_keytab') %>
```